zeta/zeta.py

```diff
@@ -155,19 +160,22 @@ def gen_creation(self):
                 # Getting persistent
                 if 'persistent' in v and v['persistent']:
                     persistent = "1"
+
                 # Getting callbacks
                 if 'subscribers' in v:
                     subscribers_list = list()
                     for s in v['subscribers']:
-                        subscribers_list.append(s['name'] +
-                                                "_service_callback")
+                        for s_k, s_v in s.items() :
```

Comment:
If you will not use the value here, it seems to be better to use .keys() instead of .items()